.github/actions/setup-coral/action.yaml

```diff
@@ -0,0 +1,49 @@
+name: Coral setup and install dependencies
+description:
+  This action installs node and pnpm in the correct version. 
+  It installs all needed dependencies and handles the pnpm store for caching. 
+  Node version and pnpm version should not be changed.
+
+inputs:
+  NODE_VERSION:
+    description: Node version for CI, matches version in .npmrc
+    default: '18'
+    required: false
+  PNPM_VERSION:
+    description: PNPM version for CI, matches version range defined in package.json
+    default: '7.13.4'
+    required: false
```

Comment:
Could there be a single source of truth for the versions? I think there's now: `.npmrc`, `package.json` and this.

Reply:
Yes, there should be. I wanted to look up what a safe way for doing this is (answers I found were a bit contradictory) but then de-prioritised it 🙈  to get this thing out. But I'll check that again later, since we want to update pnpm at some point, too, and since it's a major version update it would be important for all of us to be on the same version then (without having to coordinate it manually)

Reply:
Solved it!! :D

Based on my search-the-internet, having package.json with `engines` and `engineStrict` should make using the `.npmrc` unnecessary for us. 

The github action for setup now uses the versions defined in package.json, so we are sure we're running locally as well as on CI on the same version range. 


https://github.com/aiven/klaw/pull/931/commits/5740b7bd699e5fe0ae3ab2ad72ef049aec76ceb8

Reply:
🙌🏼 TIL, thank you